common/spl/spl.c

```diff
@@ -756,6 +756,15 @@ void board_init_r(gd_t *dummy1, ulong dummy2)
 #endif
 #endif
 
+#ifdef CONFIG_SPL_BOOTFIRMWARE_INFO
+	ret = get_boot_firmware_info();
+	if (ret) {
+		printf(SPL_TPL_PROMPT
+		       "SPL firmware version detect failed (err=%d)\n", ret);
+		hang();
```

Comment:
shouldn't the `hang` be restricted to `SPL_BOOTFIRMWARE_INFO_STRICT` ?

Reply:
this is handled in `common/bootfirmware_info.c`

if `SPL_BOOTFIRMWARE_INFO_STRICT` or `BOOTFIRMWARE_INFO_STRICT` is not set, get_boot_firmware_info() will always return `0`

Reply:
```
 37 out:                                                                            
 38         /* we should stop booting, if boot firmware version is not found */     
 39         if (CONFIG_IS_ENABLED(BOOTFIRMWARE_INFO_STRICT) && ret != 0) {          
 40                 printf("Fail to read boot firmware info from DTB, "             
 41                        "abort boot as BOOTFIRMWARE_INFO_STRICT is set\n");      
 42                 return ret;                                                     
 43         }                                                                       
 44                                                                                 
 45         return 0;                                                               
 46 }      
```

Reply:
As I didn't see any use of SPL_BOOTFIRMWARE_INFO_STRICT, that's my question. Thanks for the clarifications